features/landing-page/Navbar.tsx

```diff
@@ -0,0 +1,75 @@
+import { AuthForm } from "features/auth/AuthForm"
+import { useState } from "react"
+import { useTranslation } from "next-i18next";
+
+export default function Navbar() {
+    const [openLogin, setOpenLogin] = useState<Boolean>(false)
```

Comment:
```suggestion
    const [openLogin, setOpenLogin] = useState<boolean>(false)
```

Boolean is a constructor, not bool type 😁